Automattic/wp-calypso
# Theme Showcase: Use UpsellNudge component for Upwork banner

#### Changes proposed in this Pull Request

* Update current Upwork upsell to match existing UpsellNudges
* We may want to check in with marketing folks to make sure this change is OK from their end before we ship it.

**Before**

<img width="1355" alt="Screen Shot 2021-06-29 at 4 39 02 PM" src="https://user-images.githubusercontent.com/2124984/123864233-859bc780-d8f8-11eb-9039-2a9d826ab6f8.png">

**After**

<img width="1368" alt="Screen Shot 2021-06-29 at 4 48 21 PM" src="https://user-images.githubusercontent.com/2124984/123865380-d95ae080-d8f9-11eb-8da3-9238bf73e89a.png">

#### Testing instructions

* Switch to this PR, navigate to `/themes/[site]`
* Scroll to the bottom, click "Show all themes", you should see the new Upwork banner if you haven't dismissed it already
* Make sure the banner only appears to folks its intended to (need to double-check to see who that is)
* Make sure there are no visual bugs introduced
* Ensure notice can be dismissed
* Make sure all Tracks events (impression, click, dismiss) still work the same as before

Fixes #54087
